notifications_utils/postal_address.py

```diff
@@ -54,6 +56,19 @@ def __bool__(self):
     def __repr__(self):
         return f"{self.__class__.__name__}({repr(self.raw_address)})"
 
+    def _parse_and_remove_bfpo(self, lines):
```

Comment:
I think maybe `_parse_and_remove_bfpo_lines` is better - we are removing the bfpo lines from `self._lines` and saving the BFPO number in another variable.

Reply:
still a bit weird for me, since it's a function that that returns both the BFPO line, and the rest of the lines, but separately. If it was a "remove" function, wouldn't it return just the lines without BFPO?

Reply:
I'll change `remove` to `extract`.

> wouldn't it return just the lines without BFPO
This is what it does 😓 It removes any bfpo-related lines from the `lines` list passed in, essentialy separating all of the BFPO stuff into a separate variable.